pycuda/gpuarray.py

```diff
@@ -757,6 +767,36 @@ def view(self, dtype=None):
                 base=self,
                 gpudata=int(self.gpudata))
 
+    def transpose(self, axes=None):
+        """Permute the dimensions of an array.
```

Comment:
FYI: PyCUDA doesn't use the numpy docstring convention--it uses Sphinx's built-in one.